tests/e2e/upgrade/govexec.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/ory/dockertest/v3/docker"
 )
 
+// default upgrade info to fill necessary flag value
+const upgradeInfo = `{"binaries":{"darwin/arm64":"https://github.com/evmos/evmos/releases/download/v10.0.1/evmos_10.0.1_Darwin_arm64.tar.gz?checksum=5ab0035078f04afccfa0a6c37aed6ddea2198e02b8dd084f004c393d9d04d0f0"}}`
```

Comment:
can we use the variable from the `v10` upgrade directory?

Reply:
@fedekunze yes, just got string from @MalteHerrmann gist, client only checks that hash matches the content, so It's only mocking string to let submittion pass

Reply:
@fedekunze tried `v10` info but got this:
```shell
Error:
invalid url "https://github.com/evmos/evmos/releases/download/v10.0.0/evmos_10.0.0_Darwin_arm64.tar.gz" in binaries[darwin/arm64]: missing checksum query parameter
```

Reply:
Unfortunately it is mandatory to provide a valid link with a checksum now since SDK v0.46. This means that the `upgradeInfo` can only be correctly formed AFTER the release is already done, which is a bit of a pain.

Relevant code in the SDK is [here](https://github.com/cosmos/cosmos-sdk/blob/f771f20da49afc0fa3e0e75da431733382ddf3dd/x/upgrade/plan/downloader.go#L133-L143).

Reply:
@4rgon4ut I just found the option to add the `--no-validate` flag to the `submit-legacy-proposal` transaction command! This should enable us to NOT have to define the upgradeInfo after all!

https://github.com/cosmos/cosmos-sdk/blob/6f3380d40b3dbcaf0d04570b3a5bbf9662cbbbe6/x/upgrade/client/cli/tx.go#L101